src/test/bundles/07-helm-overrides/uds-config-invalid.yaml

```diff
@@ -0,0 +1,6 @@
+options:
+  log_level: debug
```

Comment:
We probably can't do this for `variables`, but is it possible to have it error out if `log_level` is misspelled?

Reply:
Noting that it does error out if `options` is misspelled which is 🤌

Reply:
I think that list is all the options, it'd be better to have that in a Go type though. I think today all those options exist in https://github.com/defenseunicorns/uds-cli/blob/main/src/cmd/root.go

Reply:
Added an "enum" for valid options and refactored function for unit testing